namer/web/server.py

```diff
@@ -32,21 +32,22 @@ class WebServer:
         'font/woff2': '.woff2',
     }
 
-    def __init__(self, host: str, port: int, webroot: Optional[str], blueprints: List[Blueprint]):
+    def __init__(self, host: str, port: int, webroot: Optional[str], blueprints: List[Blueprint], static_url: Optional[str] = 'public'):
         self.__host = host
         self.__port = port
         self.__webroot = webroot
         self.__path = '/' if not self.__webroot else self.__webroot
-        self.__app = Flask(__name__, static_url_path=self.__path, static_folder='public', template_folder='templates')
+        self.__app = Flask(__name__, static_url_path=self.__path, static_folder=static_url, template_folder='templates')
         self.__blueprints = blueprints
 
         self.__add_mime_types()
         self.__register_blueprints()
         self.__make_server()
+        print(self.__app.url_map)
```

Comment:
better have method like `get_url_map()`, or it's just temp for debug?

Reply:
removed